tests/functional/utils.py

```diff
@@ -0,0 +1,8 @@
+from dbt.tests.util import get_model_file, relation_from_name, set_model_file
+
+
+def update_model(project, name: str, model: str) -> str:
```

Comment:
This was copy/pasted from other adapters. We should move it to `dbt-tests-adapter` eventually, but that would complicate this PR. It should be done as a separate exercise.